test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -1172,15 +1172,15 @@ BOOST_AUTO_TEST_CASE(state_variable_accessors)
 	BOOST_REQUIRE(function && function->hasDeclaration());
 	auto returnParams = function->returnParameterTypes();
 	BOOST_CHECK_EQUAL(returnParams.at(0)->canonicalName(false), "uint256");
-	BOOST_CHECK(function->isConstant());
+	BOOST_CHECK(function->stateMutability() == StateMutability::View);
```

Comment:
These here require `View` (and cannot be substituted by `Pure`) because they ensure getters are view.